src/components/main.vue

```diff
@@ -82,16 +82,11 @@
       </div>
 
       <ResultsList v-if="$x.totalResults" v-infinite-scroll:body-scroll>
-        <template #layout="{ results }">
-          <BaseVariableColumnGrid
-            #default="{ item: result }"
-            class="x-grid"
-            :animation="resultsAnimation"
-            :items="results"
-          >
+        <BaseVariableColumnGrid class="x-grid" :animation="resultsAnimation" :items="results">
+          <template #default="{ item: result }">
             <Result :result="result" />
-          </BaseVariableColumnGrid>
-        </template>
+          </template>
+        </BaseVariableColumnGrid>
```

Comment:
This is working (I think thanks to injection) but it is throwing errors, as the `ResultsList` binds to the default slot scope `items` instead of `results`. Aside from that, I think you can use the abbreviated syntax for lone default slots  combining it with the named slots shorthand ([docs reference](https://vuejs.org/v2/guide/components-slots.html#Named-Slots-Shorthand)).

Suggestion:

```
<ResultsList v-if="$x.totalResults" #default="{ items }" v-infinite-scroll:body-scroll>
        <BaseVariableColumnGrid
          #default="{ item }"
          class="x-grid"
          :animation="resultsAnimation"
          :items="items"
        >
          <Result :result="item" />
        </BaseVariableColumnGrid>
</ResultsList>
```

Reply:
You're right, the `:items`. prop shouldn't be here, and it should just make the injection to work.